pkgs/top-level/all-packages.nix

```
@@ -32273,6 +32273,8 @@ with pkgs;
 
   ideogram = callPackage ../applications/graphics/ideogram { };
 
+  instaloader = with python3Packages; toPythonApplication instaloader;
```

```suggestion
  instaloader = python3Packages.toPythonApplication (callPackage ../tools/misc/instaloader { });
;
```

---

getting this 
```bash
$ nix-shell -I nixpkgs=/home/creator54/nixpkgs -p instaloader

error: Function called without required argument "buildPythonPackage" at /home/creator54/nixpkgs/pkgs/tools/misc/instaloader/default.nix:2
(use '--show-trace' to show detailed location information)
```

---

I didn't test before if that works but the following should:

```suggestion
  instaloader = python3Packages.callPackage ../tools/misc/instaloader { };
```